.circleci/config.yml

```diff
@@ -94,12 +94,12 @@ jobs:
       - restore_gems
       - run:
           name: Bundle install
-          command: bundle check --path $BUNDLE_PATH || bundle install --path $BUNDLE_PATH
+          command: bundle config set --local path 'vendor/bundle' && bundle install
```

Comment:
`--path` is deprecated, got this warning from cci, so changed to the recommended behavior.